service/gcs/runtime/runc/utils.go

```diff
@@ -88,3 +92,40 @@ func (r *runcRuntime) processExists(pid int) bool {
 	_, err := os.Stat(filepath.Join("/proc", strconv.Itoa(pid)))
 	return !os.IsNotExist(err)
 }
+
+func getRuncLogError(logPath string) interface{} {
+	reader, err := os.OpenFile(logPath, syscall.O_RDONLY, 0644)
+	if err != nil {
+		return nil
+	}
+	defer reader.Close()
+
+	var lastErr interface{}
+	dec := json.NewDecoder(reader)
+	for {
+		var out map[string]interface{}
```

Comment:
Ok last thing, a lot of time code looks like this `logrus.WithError(err).Error("we hit an error")`. This code would return "we hit an error" but we should also capture `err` as well.

Reply:
Done.